boxes.d/00-suse.yaml

```diff
@@ -0,0 +1,8 @@
+# using elastic's image since it is sp3, which is not offered by SUSE yet
+sles12sp3:
+  box_name: elastic/sles-12-x86_64
```

Comment:
I'd prefer not to use non-official boxes.

Reply:
as related note: also the JeOS SLES 11SP3 image is not intended for general usage.

It's a long standing problem without real solution. In the past I built SLES baseboxes with packer and distributed these over internal, non-internet-reachable HTTP servers for legal reasons.

Reply:
unfortunately, there is no SUSE supported box for SLES12-sp3. The good news is the suse supported SLES12-sp2 and SLES12-sp1 spin up fine.

Here is my proposal:
- Add SLES12 sp3 image from elastic until suse releases one, then switch to that.
- Use SUSE supported  SLES12-sp2 and SLES12-sp1 currently (updated in this PR)
- Investigate "monkey-patching" SUSEs SLES11-sp3 box by adding a vagrant user and releasing a new box to vagrant cloud, then using that box (this would be done outside of this PR)

Reply:
@mmoll what is the JeOS image? Is that what elastic's image is based on?

Reply:
yeah, but that's gone from this PR now.